packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -158,7 +158,14 @@ export class GarbageCollector implements IGarbageCollector {
 	protected constructor(createParams: IGarbageCollectorCreateParams) {
 		this.runtime = createParams.runtime;
 		this.isSummarizerClient = createParams.isSummarizerClient;
-		this.getNodePackagePath = createParams.getNodePackagePath;
+		this.getNodePackagePath = async (nodePath: string) => {
```

Comment:
@agarwal-navin - I could also put this extra logic in `ContainerRuntime` or GC Telemetry Tracker. Thoughts?

Reply:
I would put in container runtime. Ideally, GC shouldn't care abou this. Container runtime is the one that calls with "/" as the node. Also, instead of undefined, maybe pass "Root" or something so we can tell in telemetry what this is.